client/blocks/comments/form.jsx

```diff
@@ -161,7 +162,7 @@ class PostCommentForm extends React.Component {
 		switch ( error.error ) {
 			case 'comment_duplicate':
 				message = translate(
-					"Duplicate comment detected. It looks like you've already said that!",
+					"Duplicate comment detected. It looks like you've already said that!"
```

Comment:
There used to be special handling in place for duplicate comments, but the data layer no longer passes along this information to the UI. 

I've filed a separate issue for this: https://github.com/Automattic/wp-calypso/issues/18562